pyanaconda/modules/localization/localization.py

```diff
@@ -195,39 +198,27 @@ def get_layout_variant_description(self, layout_variant, with_lang=True, xlated=
 
         return _get_layout_variant_description(layout_variant, self._layout_infos, with_lang, xlated)
 
+    def get_keyboard_layouts(self):
+        """Get localized keyboard layouts
 
-    def get_locale_keyboard_layouts(self, lang):
-        """Get localized keyboard layouts for a given locale.
-
-        :param lang: locale string (e.g., "cs_CZ.UTF-8")
         :return: list of dictionaries with keyboard layout information
         """
-        language_id = get_language_id(lang)
-
-        english_name = get_english_name(language_id)
-
         # rxkb_context.layouts lists all XKB layouts, including variants and less common options,
-        # while langtable.list_keyboards filters for the most relevant layouts per language.
+        # while langtable.list_keyboards filters for the most relevant layouts.
         keyboards = self._layout_infos.items()
-        langtable_keyboards = get_locale_keyboards(language_id)
-
-        # Sort the available keyboards by name alphabetically and but put the most common ones
-        # (langtable) on top
-        keyboards = sorted(keyboards, key=lambda x: x[0])
-        keyboards = sorted(
-            keyboards,
-            key=lambda x: langtable_keyboards.index(x[0].replace(" ", "")) if x[0].replace(" ", "") in langtable_keyboards else 999
-        )
+        langtable_keyboards = get_common_keyboard_layouts()
```

Comment:
Are you sure you will get all the keyboard layouts? Based on the documentation
https://github.com/mike-fabian/langtable/blob/main/langtable/langtable.py#L2227 it should list only the highest ranked layouts.

Based on the code it will add all the layouts which has rank higher than 500
https://github.com/mike-fabian/langtable/blob/main/langtable/langtable.py#L2267

Reply:
I don't want to get all the keyboards from langtable, I use the for settings the is_common attribute. 
see:
`layout.is_common = name in langtable_keyboards and is_common_lang`

Reply:
You mean I guess rename the variable? 
s/langtable_keyboards/langtable_common_keyboards

Sure I can do that.

Reply:
I see, sorry I missed the source of the keyboard layouts is above. In that case this is fine as the code is used differently than I expected.